packages/mobile/src/screens/app-screen/AppTabScreen.tsx

```diff
@@ -180,11 +185,13 @@ export const AppTabScreen = ({ baseScreen, Stack }: AppTabScreenProps) => {
         component={CollectionScreen}
         options={screenOptions}
       />
-      <Stack.Screen
-        name='EditPlaylist'
-        component={EditPlaylistScreen}
-        options={screenOptions}
-      />
+      {!isPlaylistUpdatesEnabled ? (
```

Comment:
do we want this in the app-tab-screen and the app-screen? kinda forget where we should put it, but id imagine one spot?

Reply:
yea the two different types have different headers so they have to be in one vs the other unless im missing something